config/v201/config.json

```diff
@@ -1,56 +1,53 @@
-[
-    {
-        "name": "Connector",
-        "evse_id": 1,
-        "connector_id": 1,
-        "variables": {
-            "ConnectorAvailable": {
-                "variable_name": "Available",
-                "attributes": {
-                    "Actual": true
-                }
-            },
-            "ConnectorType": {
-                "variable_name": "ConnectorType",
-                "attributes": {
-                    "Actual": ""
-                }
-            },
-            "ConnectorSupplyPhases": {
-                "variable_name": "SupplyPhases",
-                "attributes": {
-                    "Actual": 42
-                }
+{
```

Comment:
So it seems like you made some more changes here during the rebase than intended ;)